src/main/java/com/sda/games/wheelOfFortune/methods/WheelGame.java

```diff
@@ -0,0 +1,230 @@
+    final static int fullGuessBonus = 100;
+    static int score = 0;
+    static int emptySlots;
+    static int drawnNumberCategory = generateRNCategory();
+    static int drawnNumberWord = generateRNWord(drawnNumberCategory);
+    static String guessMePhrase = idWord(drawnNumberWord);
+    final static Integer phraseLength = guessMePhrase.length();
+    final static String[] phraseKnown = prepareKnown();
+    static String[] phraseUnknown = prepareUnknown();
+
+
+    public static void startGame() {
+        //noinspection SpellCheckingInspection
+        System.out.println("Liter do zgadnięcia: " + emptySlots);
+        showMeUnknownAndAsk();
+        while (emptySlots > 0) {
+            String letter = userInputLetter();
+            //noinspection SpellCheckingInspection
+            System.out.print("Wprowadzono literę " + letter + ". ");
+            if (isLetterInKnown(letter)) {
+                if (isLetterInUnknown(letter)) { //pierwsza - "tak" - nieznana
+                    modifyUnknown(letter);
+                    if (choiceValidation()) {
+                        guessFullPhrase();
+                    }
+                } /*else {
+                    System.out.println("(do usuniecia: Litera już wcześniej została dodana do wyrażenia. Punktów: " + score);
+                }*/
+            }
+        }
+        //noinspection SpellCheckingInspection
+        System.out.println("Gra zakończona: hasło odgadnięte. Liczba punktów:  " + score);
+    }
+
+
+    private static String idWord(int nr) {
+        WordsDao wordsDao = new WordsDao(WheelGame.hibernateFactory);
+        return wordsDao.getById(nr).getWord();
+    }
+
+    private static String[] prepareKnown() {
+        return guessMePhrase.split("");
+    }
+
+    public static String[] prepareUnknown() {
+        String[] phraseUnknown = new String[phraseLength];
+        for (int i = 0; i < phraseLength; i++) {
+            if (phraseKnown[i].equals(" ")) {
+                phraseUnknown[i] = " ";
+            } else {
+                phraseUnknown[i] = "_";
+                emptySlots = emptySlots + 1;
+            }
+        }
+        return phraseUnknown;
+    }
+
+    public static void modifyUnknown(String letter) {
+        for (int i = 0; i < phraseLength; i++) {
+            if (phraseKnown[i].toUpperCase(Locale.ROOT).equals(letter)) {
+                phraseUnknown[i] = letter;
+                emptySlots = emptySlots - 1;
+                score = score + basicPointValue;
+            }
+        }
+        showMeUnknown();
+        //noinspection SpellCheckingInspection
+        System.out.println("score = " + score + " do zgadnięcia " + emptySlots);
+    }
+
+    public static void showMeUnknownAndAsk() {
+        //noinspection SpellCheckingInspection
+        System.out.println("Zgadnij hasło, Category: " + categoryDao.getById(generateRNCategory()).getName());
+        showMeUnknown();
+    }
+
+    public static void showMeUnknown() {
```

Comment:
po samych nazwach funkcji mozecie sprobowac posegregowac ten kod na rozne klasy :D ja np widze czesto unnknown wiec pewnie zrobil bym jakas klase zwiazana z tym, jak jest unknown to zrobil bym pewnie jakas do known, czesto pojawia sie tez letter wiec jakas klasa do obslugi liter :)